charts/scalardl/templates/ledger/certmanager.yaml

```diff
@@ -0,0 +1,83 @@
+{{- if .Values.ledger.tls.certManager.enabled }}
+{{- if not .Values.ledger.tls.certManager.issuerRef }}
```

Comment:
> Is there a request from a user who wants to use cert-manager while using an arbitrary issuer?

Yes.

I interviewed the business team about some existing customers' and potential customers' requirements.

As a result, there are some users who want to use the cert-manager with an arbitrary issuer. (i.e., There are some users who don't want to use self-signed certificate for some security reasons.)

> If not, I feel that it would be better to use selfsigned when using the cert-manager function of this Helm Chart, and not use it at all when using an arbitrary certificate.

Ah, I see. Thank you for your suggestion.

I thought there are some users who want to use self-signed certificate. For example, users who have no problem with self-signed certificate or developers who want to test TLS features.

This is a reason way I remained the `self-signed CA` pattern.

However, if they want to use self-signed CA, it is easy to deploy self-signed CA (self-signed issuer) follow the [official docuemnt of cert-manager](https://cert-manager.io/docs/configuration/selfsigned/).

So, now, I'm thinking about it's fine to remove self-signed CA pattern.

Do you have any concerns when we support both `use automatically deployed self-signed CA (self-singed issuer)` and `use arbitrary issuer pattern`?